app/utils/AMQP.js

```diff
@@ -0,0 +1,61 @@
+const amqplib = require('amqplib');
+
+const RESERVATION_REQUEST_QUEUE = 'rrq';
```

Comment:
Better to avoid abbreviations because the queue name can be used in monitoring service